analyses/molecular-subtyping-embryonal/00-embryonal-select-pathology-dx.Rmd

```diff
@@ -0,0 +1,177 @@
+
+1. An RNA-seq biospecimen sample includes a _TTYH1_ fusion (5' partner) [per this comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/401#issuecomment-573669727).
+2. Any sample with "Embryonal tumor" in the `broad_histology` column of the metadata `pbta-histologies.tsv` that is not labeled "Medulloblastoma" or "Atypical Teratoid Rhabdoid Tumor (ATRT)" in `pathology_diagnosis` or `integrated_diagnosis` columns [per this comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/251#issuecomment-568220913).
+
+In an upcoming release, `integrated_diagnosis`, which can be updated as the result of subtyping, will be used to populate the `broad_histology` column (see [#748](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/748)).
+Thus, molecular subtyping modules need to be upstream of `broad_histology` and use the `pathology_diagnosis` and `pathology_free_text_diagnosis` fields.
+
+Filtering on the basis of `broad_histology == Embryonal tumor` is more straightforward than using the pathology diagnosis fields, so we include this notebook to put together the terms in `pathology_diagnosis` and `pathology_free_text_diagnosis`.
+
+We will use the 2016 WHO Classification as our guide ([Louis et al. _Acta Neuropathol._ doi: 10.1007/s00401-016-1545-1](10.1007/s00401-016-1545-1)) and take a look at the current version of the histology file (`release-v17-20200908`).
+
+## Set up
+
+```{r}
+library(tidyverse)
+```
+
+### Directories and files
+
+We're going to tie this to a specific release.
+
+```{r}
+data_dir <- file.path("..", "..", "data", "release-v17-20200908")
+histologies_file <- file.path(data_dir, "pbta-histologies.tsv")
+```
+
+We're going to save the pathology diagnosis information we'll use to generate the subset files in a directory `hgg-subset`.
+
+```{r}
+output_dir <- "subset-files"
+output_file <- file.path(output_dir, 
+                         "embryonal_subtyping_path_dx_strings.json")
+```
+
+## Read in data
+
+```{r}
+histologies_df <- read_tsv(histologies_file)
+```
+
+## Explore the pathology diagnoses
+
+### `broad_histology == Embryonal tumor`
+
+In the current histologies file, if we filter based on `broad_histology` as we did originally, and drop medulloblastoma and ATRT samples using the pathology diagnosis lables, what is left in the pathology diagnosis fields?
+Note that some of the `broad_histology` values will have been altered based on earlier subtyping efforts.
+
+```{r}
+histologies_df %>% 
+  filter(broad_histology == "Embryonal tumor",
+         !(pathology_diagnosis %in% c("Medulloblastoma",
+                                   "Atypical Teratoid Rhabdoid Tumor (ATRT)"))) %>%
+  count(pathology_diagnosis) %>%
+  arrange(desc(n))
+```
+
+For the most part, this is as we would expect given the 2016 WHO classifications.
+There is an ependymoma sample and a high-grade glioma sample that should probably be excluded.
```

Comment:
Notably - the EPN sample appears to be changed to embryonal due to the presence of an _MN1_ fusion https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/785#issuecomment-695015488. We only look at _TTYH1_ fusions in `analyses/molecular-subtyping-embryonal/01-samples-to-subset.Rmd` so this may be a good opportunity to update that notebook to include samples on the basis of the presence of _MN1_ fusions.

Reply:
@jharenza wanted to bring your attention to this comment specifically - should we update the step where we include samples (looking at the entire cohort) on the basis of the presence of fusions to include _MN1_ fusions and _TTYH1_ fusions (only the latter is addressed currently)? I suspect yes - I can not see why this EPN sample would be included otherwise.

Reply:
Sorry, I missed this, but yes to this:

>should we update the step where we include samples (looking at the entire cohort) on the basis of the presence of fusions to include MN1 fusions and TTYH1 fusions (only the latter is addressed currently)?